internal/pkg/filesystem/filesystem.go

```diff
@@ -139,8 +140,7 @@ func IsFrom(path, base string) bool {
 	if base == "" || base == "." {
 		return true
 	}
-	baseWithSep := base + string(PathSeparator)
-	return strings.HasPrefix(path, baseWithSep)
+	return strings.HasPrefix(strings.TrimRight(path, PathSeparatorString), base)
```

Comment:
`base + ` ... alokuje novy string, ... spusta sa to pre kazdy objekt, takze je to velmi vela volani, ak je vela konfiguracii a config rows.

`TrimRight` iba oreze povodny string, takze sa nova pamat nealokuje.